src/main/groovy/edu/oregonstate/mist/osuevents/resources/CSVHelperFunctions.groovy

```diff
@@ -0,0 +1,23 @@
+package edu.oregonstate.mist.osuevents.resources
+
+import java.time.ZoneId
+import java.time.ZonedDateTime
+import java.time.format.DateTimeFormatter
+
+class CSVHelperFunctions {
+    public static String getCSVDate(ZonedDateTime dateTime, ZoneId backendTimeZone) {
+        DateTimeFormatter csvDateFormatter = DateTimeFormatter
+                .ofPattern("MM/dd/yyyy")
+                .withZone(backendTimeZone)
+
+        dateTime.format(csvDateFormatter)
+    }
+
+    public static String getCSVTime(ZonedDateTime dateTime, ZoneId backendTimeZone) {
```

Comment:
These two methods are not DRY. How about having a private helper method. It would take the two parameters that these two methods already accept, but also take in a pattern string. Both of these methods would then just call the private method and return the output right away.